packages/core/src/styles/themes/dark-theme.stories.mdx

```diff
@@ -0,0 +1,180 @@
+
+<Meta title="Foundation/Themes/Dark Theme" />
+
+# Dark Theme
+
+Clarity provides a built in dark theme which can be loaded and dynamically enabled
+in the browser.
+
+```scss
+// import into sass or css
+@import '~@clr/core/styles/theme.dark.min.css';
+
+// or import in html
+<link href="/node_modules/@clr/core/styles/theme.dark.min.css" rel="stylesheet">
+```
+
+```html
+<body cds-theme="dark">
+  ...
+</body>
+```
+
+## Accordion
+
+<Preview>
+  <Story id="components-preview-accordion-stories--dark-theme" />
+</Preview>
+
+## Alerts
+
+<Preview>
+  <Story id="components-alert-group-stories--dark-theme" />
+</Preview>
+
+### Badge
+
+<Preview>
+  <Story id="components-badge-stories--dark-theme" />
+</Preview>
+
+## Tag
+
+<Preview>
+  <Story id="components-tag-stories--dark-theme" />
+</Preview>
+
+## Button
+
+<Preview>
+  <Story id="components-button-stories--dark-theme" />
+</Preview>
+
+## Icon Button
+
+<Preview>
+  <Story id="components-icon-button-stories--dark-theme" />
+</Preview>
+
+## Icon
+
+<Preview>
+  <Story id="components-icon-stories--dark-theme" />
+</Preview>
+
+## Circular Progress
+
+<Preview>
+  <Story id="components-preview-circular-progress-stories--dark-theme" />
+</Preview>
+
+## Modal
+
+<Preview>
+  <Story id="components-modal-stories--dark-theme" />
+</Preview>
+
+## Forms
+
+<Preview>
+  <Story id="forms-preview-forms-stories--dark-theme" />
+</Preview>
+
+## Checkbox
+
+<Preview>
+  <Story id="forms-preview-checkbox-stories--dark-theme" />
+</Preview>
+
+## Datalist
+
+<Preview>
+  <Story id="forms-preview-datalist-stories--dark-theme" />
+</Preview>
+
+## Date
+
+<Preview>
+  <Story id="forms-preview-date-stories--dark-theme" />
+</Preview>
+
+## File
+
+<Preview>
+  <Story id="forms-preview-file-stories--dark-theme" />
+</Preview>
+
+## Input
+
+<Preview>
+  <Story id="forms-preview-input-stories--dark-theme" />
+</Preview>
+
+## Password
+
+<Preview>
+  <Story id="forms-preview-password-stories--dark-theme" />
+</Preview>
+
+## Radio
+
+<Preview>
+  <Story id="forms-preview-radio-stories--dark-theme" />
+</Preview>
+
+## Range
+
+<Preview>
+  <Story id="forms-preview-range-stories--dark-theme" />
+</Preview>
+
+## Search
+
+<Preview>
+  <Story id="forms-preview-search-stories--dark-theme" />
+</Preview>
+
+## Select
+
+<Preview>
+  <Story id="forms-preview-select-stories--dark-theme" />
+</Preview>
+
+## Textarea
+
+<Preview>
+  <Story id="forms-preview-textarea-stories--dark-theme" />
+</Preview>
+
+## Time
+
+<Preview>
+  <Story id="forms-preview-time-stories--dark-theme" />
+</Preview>
+
+## Toggle
+
+<Preview>
+  <Story id="forms-preview-toggle-stories--dark-theme" />
+</Preview>
+
+## Typography
+
+<Preview>
+  <Story id="foundation-typography-stories--dark-theme" />
+</Preview>
+
+## Divider
+
+<Preview>
+  <Story id="components-preview-divider-stories--dark-theme" />
+</Preview>
+
+## List
+
+<Preview>
+  <Story id="components-list-stories--dark-theme" />
+</Preview>
```

Comment:
Each component has a new dark theme story which is then added here and each independent component mdx file.